source/main.cpp

```diff
@@ -129,7 +129,7 @@ int main(int argc, char **argv) {
 	int patchfile = 0;
 	int header;
 	rawDataOffset=0;
-	char datName[]="YS:/" DATNAME;
+	char datName[MAX_DATNAME_LEN]="YS:/" DATNAME;
```

Comment:
@Syphurith My codes put path string at 0x11C,with zero padding up to offset 0x14F at any situation.